source/services/MonitoringService.ts

```diff
@@ -0,0 +1,32 @@
+import { ComponentType } from "react";
+import env from "react-native-config";
+
+import SentryMonitoringService from "./SentryMonitoringService";
+
+export interface MonitoringService {
+  init: () => void;
+  sendError: (Error: Error | string | null) => void;
+  wrap: (component: ComponentType<unknown>) => ComponentType<unknown>;
+}
```

Comment:
_Maybe_ move interface to a separate file, to separate definitions from declarations (might be overkill here).

Reply:
moved to separate folder and file